### PR TITLE
Add official material-theme for Hyper

### PR DESCRIPTION
**Pull request by equinusocio**
- [x] The title for my package or theme uses its npm title (`hyper-plugin-example`)
- [x] The link for my package or theme uses the npmjs.com link (https://www.npmjs.com/package/hyper-material-theme)
- [x] Put your awesome item at the **BOTTOM** of the correct (plugin, theme, or resource) list.
- [x] There is a visual representation of what my plugin or theme does in the repo. (Plugin: screenshot or gif || Theme: screenshot of the theme applied to Hyper)
- [x] **VERY IMPORTANT:** I've written a short (one sentence) description for my package or theme as the description of the awesome plugin, theme, or resource in the README.md file I'm submitting in the PR.

This is the official [Material Theme](https://github.com/equinusocio/material-theme) porting for Hyper.

### CODE REVIEW

**Comment by equinusocio** [comment]
What's the problem..

**Comment by iamstarkov** [comment]
@equinusocio no worries

File: README.md [hunk 1]
```diff
@@ -126,4 +126,4 @@ Like `awesome-hyper`? Reach out to [@bitandbang](https://twitter.com/bitandbang)
 * [hyper-sierra](https://www.npmjs.com/package/hyper-sierra) - A dark theme for Hyper inspired by the Sierra Nevada.
 * [hyper-sierra-vibrancy](https://www.npmjs.com/package/hyper-sierra-vibrancy) - hyper-sierra with vibrancy.
 * [hyper-thirtyone](https://www.npmjs.com/package/hyper-thirtyone) - A slightly modified classic linux terminal theme for Hyper.
-
+* [hyper-material-theme](https://www.npmjs.com/package/hyper-material-theme) - Dark - The most epic material theme now on Hyper App.
```
Comment (iamstarkov) [inline README.md]: can you keep one end line?

**Comment by equinusocio** [comment]
@iamstarkov done

File: README.md (hunk 1 above)
Comment (equinusocio) [inline README.md]: done

**Comment by equinusocio** [comment]
@iamstarkov Any update?

**Comment by bnb** [comment]
@equinusocio I have fixed the merge conflict and committed 😊 Test passed, merged! 🚀

**Comment by iamstarkov** [comment]
yay

**Pull request merged**